pkg/beholder/pb/keystone/keystone_custom.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/beholder/pb/keystone";
+
+package pb;
+
+message KeystoneCustomMessage {
```

Comment:
For the required Keystone labels (workflowOwner, id, etc), are these expected to be added to the labels map?

Reply:
Yes, thats the expectation. I prefer the flexibility and scalability of this approach (a generic labels map) over an explicit definition of each field like I had before.